### ISSUE TITLE
running on raspberry pi os lite

### ISSUE THREAD

**Issue by maxsimongt3** [inventree/InvenTree#1334]
I'm trying to have a raspberry pi act as a server on a local network with raspberry pi os lite 32bit,
I go through the steps in the documentation but when I get to the 
inv migrate
step it tells me 
ImportError: libopenjp2.so.7: cannot open shared object file: No such file or directory

can anyone help???

**Comment by eeintech** [comment]
Have you searched on Google?

A simple search leads to this solution:
https://stackoverflow.com/questions/48012582/pillow-libopenjp2-so-7-cannot-open-shared-object-file-no-such-file-or-directo

**Comment by maxsimongt3** [comment]
Your right Francois
I ran sudo apt-get install libopenjp2-7
that wasn't enough so I also ran 
sudo apt-get install libtiff5
then it told me I needed Cairo so I went to 
https://www.cairographics.org/download/
where I found sudo apt-get install libcairo2-dev
then it told me I needed libpango1.0-0
I googled it and found https://ubuntu.pkgs.org/18.04/ubuntu-universe-amd64/libpango1.0-0_1.40.14-1_amd64.deb.html
and I ran sudo apt-get install libpango1.0-0
then I did inv migrate and I'm up and running on raspberry pi lite.

thanks for you help I'm really new to coding and command line.
I really appreciate your patience.
